pkgs/development/ruby-modules/gem-config/default.nix

```diff
@@ -211,12 +211,12 @@ in
     '';
   };
 
-  pg_query = attrs: lib.optionalAttrs (attrs.version == "2.0.1") {
+  pg_query = attrs: lib.optionalAttrs (attrs.version == "2.0.2") {
     dontBuild = false;
     postPatch = ''
       sed -i "s;'https://codeload.github.com.*';'${fetchurl {
-        url = "https://codeload.github.com/lfittl/libpg_query/tar.gz/13-2.0.0";
-        sha256 = "0ghk0dlmrn634p3zjr41fy4ipgw8i44f67a4l8cspqg0395m3rp5";
+        url = "https://codeload.github.com/lfittl/libpg_query/tar.gz/13-2.0.2";
```

Comment:
```suggestion
        url = "https://codeload.github.com/lfittl/libpg_query/tar.gz/13-${attrs.version}";
```

Reply:
These version numbers don't automatically match, historically, so I don't think this change is necessarily desirable. One is the version number for a ruby gem, and the other is for the C library it wraps.

Reply:
Oh, I thought they are the same.

Reply:
So did I, the first time I saw it! :-)

Thanks for merging.